bin/pre-commit.sh

```diff
@@ -20,12 +22,16 @@ fi
 
 # Lint staged .scss files
 scss_files=$( git diff --diff-filter=d --staged --name-only | grep -E '/*\.scss$' )
-if [ -n "$scss_files" ]; then
+if [ -n "$scss_files" ]
+	then
 	npm run lint:css:files $scss_files
 	if [ $? != 0 ]; then
 		exit 1
 	fi
 fi
 
 # Lint package.json
-npm run lint:pkg-json
+if [ -n "$( git diff --diff-filter=d --staged --name-only | grep -E 'package.json' )" ]
+	then
+	npm run lint:pkg-json
```

Comment:
If there's no change to `package.json`, no need to lint it in the pre-commit hook